src/components/CoverV2/Cover.tsx

```diff
@@ -190,9 +197,19 @@ export const Cover = ({
   fullHeight = false,
   ...props
 }: CoverProps) => {
+  const defaultImage =
+    images?.desktop ||
+    images?.tablet ||
+    images?.mobilePortrait ||
+    images?.mobileLandscape
```

Comment:
I get what's happening here and tried to find other ways to do it, but I'm surprised there don't really seem to be smarter ways to do this 😅

Reply:
yeah it's kinda annoying. I also thought about a lot of different approaches, for example adding an extra prop for a default image or if you only want to use a single image like the old cover, but that is something I want to prevent because I want to make it super clear what prop is responsible for the cover image.